lib/workload/components/sfn-nf-batch-ready-event-handler/step_functions_templates/launch_nextflow_pipeline_template.asl.json

```diff
@@ -165,6 +165,7 @@
               "workflowVersion.$": "$.workflow_inputs.workflowVersion",
               "workflowRunName.$": "$.workflow_inputs.workflowRunName",
               "timestamp.$": "$$.State.EnteredTime",
+              "status.$": "${__event_status__}",
```

Comment:
`${__event_status__}` will be static (and not a json path) so should be `"status"` not `"status.$"`

Reply:
Yup. Updated with [2f9eb43](https://github.com/umccr/orcabus/pull/667/commits/2f9eb43bcad647ec1d8c471b19c6b0024c0f2254)